docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -1067,15 +1067,3 @@ spec:
   purpose: Informational
   tags: inventory
   contributors: lucasmrod,sharon-fdm,zwass
```

Maybe instead of deleting, try without the trailing `---`?
```suggestion
  contributors: lucasmrod,sharon-fdm,zwass
---
apiVersion: v1
kind: query
spec:
  name: List osquery table names
  platform: darwin, linux, windows
  description: List all table names in the schema of the currently installed version of osquery
  query: SELECT DISTINCT name FROM osquery_registry;
  purpose: Informational
  tags: fleet, osquery, table, schema
  contributors: nonpunctual
```

---

@nonpunctual going to go ahead and commit this change to see if that fixes it; will revert to what you had if not.